scripts/gitlab/build-unix.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
```

Comment:
I think we should have ```#!/usr/env bash``` here instead

Reply:
Agreed. Need to be fixed in master though